scripts/create-super-user

```diff
@@ -9,3 +9,7 @@ ADMIN_PASSWORD_ENCODED='$2y$10$snpZ8bgdkh2hy8lDtyHF7ejD5.K1vsMqaFteCkmBhdBQj3JTl
 ADMIN_UUID='2754daf8-fea0-4bf4-af60-810764f24d71'
 
 docker exec packit-packit-db create-super-user --email "$ADMIN_EMAIL" --password "$ADMIN_PASSWORD_ENCODED" --uuid "$ADMIN_UUID"
+
+echo "Log in with":
+echo "  username: resideUser@resideAdmin.ic.ac.uk"
```

Comment:
Could use $ADMIN_EMAIL for this I guess!

Reply:
Good idea. Though looking at this I am surprised that encoded password works properly - should it not use the new secret that I added in the config?